modules/virt-cloning-vm-snapshot.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * virt/backup_restore/virt-backup-restore-snapshots.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="virt-cloning-vm-snapshot_{context}"]
+= Cloning a VM by using the web console
+
+You can create a new VM by cloning either an existing VM or a VM snapshot.
```

Comment:
I think the sentence should be changed a bit to something like
`You can create a new VM either by cloning an existing vm or using a VM snapshot as a source`
since with current phrasing the cloning verb is still on the vm snapshot